sdk/python/scilog/snippet.py

```diff
@@ -16,12 +16,11 @@ def typechecked_call(obj, *args, **kwargs):
                 )
             if len(func.__closure__) > 0:
                 property_name = func.__closure__[0].cell_contents.strip("_")
-                if obj._deprecated.get(property_name):
-                    warnings.warn(
-                        f"{property_name} is deprecated by {obj._deprecated_by[property_name]}"
-                    )
+                # if obj._deprecated.get(property_name):
```

Comment:
why did you remove the deprecation? If this was on purpose, we could remove the warning "catcher" from send_logbook_message.

Reply:
Reenabled

Reply:
I think instead of commenting this, we could use logger.log(...) instead of warning.warn so that it is less intrusive. I'd keep a log as it is indeed something that the python users want to be informed

Reply:
or logger.warning:
https://stackoverflow.com/questions/9595009/warnings-warn-vs-logging-warning

Reply:
LGTM, logging instead of warning